nullaway/src/main/java/com/uber/nullaway/fixserialization/location/AbstractSymbolLocation.java

```diff
@@ -51,9 +53,11 @@ public AbstractSymbolLocation(ElementKind type, Symbol target) {
             + ".");
     this.type = type;
     this.enclosingClass = castToNonNull(ASTHelpers.enclosingClass(target));
-    this.uri =
+    // URIs for classes are still serialized. Can be revisited in follow up later.
```

Comment:
This is not at all a clear message. What does "URIs for classes" mean in this context?
```suggestion
    // We currently serialize the URI for the classfile if the URI for the sourcefile is not available,
    // but only if said URI corresponds to a "file:" or "jimfs:" scheme (i.e. not "jar:"). It's likely that 
    // this is no longer needed and should be removed as a follow up:  
    // https://github.com/uber/NullAway/issues/716
    // Leaving this workaround up temporarily for the sake of experiments with version 
    // `1.3.6-alpha-N` of the auto-annotator.
```